css/components/base.css

```diff
@@ -1,7 +1,7 @@
 /* base styling */
 
 h1, h2, h3, h4, h5, h6, .heading {
-  line-height: 3.5rem;
```

Comment:
This can be bad for accessibility for some users that try to zoom in/out on their screens.

You should set line-height as a ratio, something like 1.3, or 1.5 rather than 3.5rem or 42px etc

Reply:
the change here is setting the value to '1.1'